app/scripts/services/Room.js

```diff
@@ -4,7 +4,10 @@
     var rooms = $firebaseArray(ref);
       
     return {
-        all: rooms
+        all: rooms,
+        createRoom: function(newRoomName) {
+            rooms.$add({name: newRoomName});        
```

Comment:
I assume you liked using the `name` key better then `$value`?  You probably figured this out already, but you should delete the older rooms if you move forward with this since they won't have the `name` property.